### ISSUE TITLE
For ActiveRecord - add current context to the prompt for hints

### ISSUE THREAD

**Issue by francis** [BoxcarsAI/boxcars#28]
When using a Boxcar::ActiveRecord boxcar from an Active Record controller, it would be nice to know context like the current_user, etc. These can be added to the Prompt.

**Comment by princetechs** [comment]
Yes, I am planning to add profile-based memory stores, which can solve these.
